tests/unit/cli/admin/test_metadata.py

```diff
@@ -688,3 +688,330 @@ def test_metadata_update_full_upload_and_run_ceremony(
                 "task_id", test_context["settings"], "Metadata Update status: "
             )
         ]
+
+
+class TestMetadataSign:
+    """Test the Metadata Sign Interaction"""
+
+    def test_metadata_sign_start(self, client, test_context):
```

Comment:
What does that check? I am not sure I see it.

Reply:
It checks the `metadata.sign` invokes the expected subcommand. 
We have it for all the subcommands.